linker_scripts/object_script.txt

```diff
@@ -352,6 +352,43 @@ D_06014860 = 0x06014860;
 /* z_en_jc_mato */
 D_06000390 = 0x06000390;
 
+/* z_en_ma_yto */
+D_06007E28 = 0x06007E28;
+D_06003D54 = 0x06003D54;
+D_0600A174 = 0x0600A174;
+D_060070EC = 0x060070EC;
+D_06003D54 = 0x06003D54;
+D_06015C28 = 0x06015C28;
+D_06005430 = 0x06005430;
+D_0600AF7C = 0x0600AF7C;
+D_06014AD8 = 0x06014AD8;
+D_06014ED8 = 0x06014ED8;
+D_060152D8 = 0x060152D8;
+D_060156D8 = 0x060156D8;
+D_06001FD0 = 0x06001FD0;
+D_06014AD8 = 0x06014AD8;
+D_06014ED8 = 0x06014ED8;
+D_060152D8 = 0x060152D8;
+D_060156D8 = 0x060156D8;
+D_06011AD8 = 0x06011AD8;
+D_060122D8 = 0x060122D8;
+D_06012AD8 = 0x06012AD8;
+D_060132D8 = 0x060132D8;
+D_06013AD8 = 0x06013AD8;
+D_060142D8 = 0x060142D8;
+D_0600A174 = 0x0600A174;
+D_0600AF7C = 0x0600AF7C;
+D_06000CC0 = 0x06000CC0;
+D_06016720 = 0x06016720;
+D_06005314 = 0x06005314;
+D_060093E8 = 0x060093E8;
+D_06007E28 = 0x06007E28;
+D_060070EC = 0x060070EC;
+D_06003D54 = 0x06003D54;
+D_06001FD0 = 0x06001FD0;
+D_060030B4 = 0x060030B4;
+D_06004370 = 0x06004370;
```

Comment:
You would know a lot more than I would since you've spend so much time with ZAPD, but does it make it easier to do object decomp if these symbols/addresses are sorted within their blocks?

Reply:
I don't really check object_script.txt much when doing object decomp, I think the symbols in the C file are more useful. I usually just check the object_script.txt to remove the symbols that are not needed anymore.